cibuildwheel/__main__.py

```diff
@@ -161,7 +161,7 @@ def main() -> None:
     # This needs to be passed on to the docker container in linux.py
     os.environ['CIBUILDWHEEL'] = '1'
 
-    if not os.path.exists(os.path.join(package_dir, 'setup.py')):
+    if not detect_setup_pyproject_existence(package_dir):
```

Comment:
Since there's now also `setup.cfg`, maybe just `is_python_project`?

Reply:
perhaps `is_python_package` since it's more descriptive?

Reply:
Even better, yes! Just thought it could be independent from the actual file names.